### PR TITLE
Track fridge raiders and employers form using unit tested library

### LINKED ISSUE #642: fridge-raiders
landing pages for fridge raiders competition

### PR DESCRIPTION
**Pull request by pitchinvasion**
This uses a module called track-form.js to put the data in local
storage. It's hacky but it's the only way to do it, we promise (signed
Leo, Dan, Mary)

Closes #642

### CODE REVIEW

**Comment by roidriscoll** [comment]
:shipit:

File: source/employers/thank-you.html.haml [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-    analytics.alias(properties.email);
-    analytics.identify(properties.email, properties);
-    window.localStorage.removeItem("employerAnalyticsProperties");
-  }
-
+  function submitFormDataToAnalytics (formData) {
+    analytics.alias(formData.email);
+    analytics.identify(formData.email, formData)
+    analytics.track("Submitted Employers Enquiry Form")
+
```
Comment (spike01) [inline source/employers/thank-you.html.haml]: Missing closing `}`?

File: source/javascripts/email_capture.js [hunk 2]
```diff
@@ -11,10 +11,12 @@
    e.g. <input type="email" data-segments="women,entrepeneurs">
  */
 
+//= require vendor/jquery
```
Comment (spike01) [inline source/javascripts/email_capture.js]: Does this do anything in this context?

**Comment by spike01** [comment]
Um, ok? If it works I guess it works, although having to use a global function per page is.. yeah. 

If email-capture is no longer being used, it should probably be fully removed (the only visible use is the line deleted from all.js)

:shipit:

**Pull request merged**